paper/user_study.tex

```diff
@@ -0,0 +1,66 @@
+Our user study is mainly based on the qualitative research.
+We conduct several interviews with the professors and students as potential customers.
+We collect feedbacks from the customers, improve user experiences and fix bugs according to the feedback.
+We also use the survey to get interviewees' opinions during the user study.
```

Comment:
What do you mean with that sentence? A survey is more the thing we did for demographic quesions
```suggestion
We also use a survey to get interviewees' opinions during the user study.
```